lib/metadataTypes/TransactionalMessage.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+const TYPE = require('../../types/mcdev.d');
+const MetadataType = require('./MetadataType');
+const Util = require('../util/util');
+
+/**
+ * TransactionalMessage MetadataType
+ *
+ * @augments MetadataType
+ */
+class TransactionalMessage extends MetadataType {
+    static subType = '';
```

Comment:
reason to set this  as empty?

Reply:
So that intellisense works. Will add a code comment for it